030_il_L216/030_il_L216_safe.py

```
@@ -0,0 +1,483 @@
+        mprotect.argtypes = [ctypes.c_void_p, ctypes.c_ulong, ctypes.c_int]
+
+        PROT_READ = 1
+        PROT_WRITE = 2
+        PROT_EXEC = 4
+
+        buf_p = ctypes.c_char_p(code)
+        size = len(code)
+        addr = valloc(size)
+
+        if not addr:
+            raise MemoryError("Failed to allocate memory")
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)